themes/src/main/resources/theme/base/login/login-otp.ftl

```diff
@@ -1,5 +1,5 @@
 <#import "template.ftl" as layout>
-    <@layout.registrationLayout; section>
+    <@layout.registrationLayout displayMessage=!messagesPerField.existsError('totp'); section>
```

Comment:
@vmuzikar I think that this is related to the fact that @mabartos was working on the error alerts in general - across all the login screens. Therefore he made the change at once. @mabartos correct me if I'm wrong.

Reply:
@Pepo48 Yes, you're right. This follow-up task was made a while after the main task, which turn on the PF4 and we were working on these issues mostly separately. And yes, we could connected it. But at this time, I think, it's only minor thing and it'll take some time to remade it. Unfortunately, we want it to merge ASAP. WDYT @vmuzikar ?

Reply:
Ok, I'm fine with that. I'm just starting to be a little bit concerned because there are too many parallel PRs, all touching similar thing, all need to work together, sometimes even touch the same files. So let's hope it'll all work together well once merged. :)